src/Application/test/MultiplyCalculatorGTest.cpp

```diff
@@ -0,0 +1,56 @@
+///////////////////////////////////////////////////
+/**
+ * @file MultiplyCalculatorGTest.cpp
+ * @brief Unit tests for MultiplyCalculator
+ */
+///////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include<limits>
+
+#include "CalculatorApplication.h"
+#include "CalculatorApplicationFactory.h"
+#include "CalculatorStrings.h"
+#include "MultiplyCalculator.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingMultiplyCalculator : public :: testing :: Test
```

Comment:
If you aren't using the fixture... just remove this class and then change all of your `TEST_F(...)` to `TEST(...)`